### ISSUE TITLE
Translation -- implement translated content on Janis

### ISSUE THREAD

**Issue by ifsimicoded** [cityofaustin/techstack#113]
- [x] all content stored in CMS --> @benweatherman 
- [x] all content NOT stored in CMS (static content) --> defined and blocked by #112
- [x] translation dynamic switching / language selection
- [x] translation persistence
- [x] language in routing for page sharing/bookmarking
- Make sure there is a polyfill for react-intl for all browsers we want to support https://caniuse.com/#search=Intl

Blocked by #111, #112 
Related/duplicate to #133 
Related/duplicate to cityofaustin/janis#111 
Related/duplicate to  cityofaustin/janis#152

**Comment by mateoclarke** [comment]
@ifsimicoded Just a heads up that we need the translated content for the 5 service pages available on the staging site by 3/9 for user testing. The content team (cc/ @courtneyjacinic) is testing the page content for comprehension with residents. However is easiest to accomplish that is good with them.

**Comment by cthibaultatx** [comment]
@ashleecoa @courtneyjacinic @LauraTrujillo is this the task for putting the translated content into janis?

**Comment by mateoclarke** [comment]
@cthibaultatx This is the task for the dev team to put the hard coded content into Janis. We're not worrying about the editing experience yet.

**Comment by ifsimicoded** [comment]
Found this to help us manage static translated content in the future. I won't be implementing this for the current ticket.
https://www.npmjs.com/package/react-intl-translations-manager

**Comment by ifsimicoded** [comment]
- Infrastructure for handling translation of static content on janis is completed via PR https://github.com/cityofaustin/janis/pull/165
- Implementation of translated content will be completed via another PR when the content is provided via issue #112

**Comment by mateoclarke** [comment]
I think these links got pasted into the wrong GH Issue so I'm putting them here instead:
- Spanish: https://drive.google.com/open?id=1F-MInnVSWD-U_YJNeDthLxC-2GWwuuy3
- Arabic: https://drive.google.com/file/d/1A2Ssy8OT6EKiGSLYhTFHYbAq6maAKt0x/view?usp=sharing
- Vietnamese: https://drive.google.com/file/d/1wobtUirII4fJWzta0-dHm_pT7DTXA_YW/view?usp=sharing

**Comment by mateoclarke** [comment]
All translations in 1 doc:
https://docs.google.com/spreadsheets/d/1go0pFgKJSryvfCUmSAyt_ylW9IGDxDhVBMvttrA4HjM/edit#gid=195265087
